dbt_copilot_helper/templates/addons/env/aurora-postgres.yml

```diff
@@ -151,3 +152,309 @@ Resources:
       FromPort: 5432
       ToPort: 5432
       SourceSecurityGroupId: !Ref EnvironmentSecurityGroup
+
+  # Enable ingress from Lambda Functions.
+  {{ service.prefix }}LambdaIngress:
+    Metadata:
+      'aws:copilot:description': 'Allow ingress from Lambda functions in my application to the DB cluster'
+    Type: AWS::EC2::SecurityGroupIngress
+    Properties:
+      Description: Ingress from Lambda Function
+      GroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+      IpProtocol: tcp
+      FromPort: 5432
+      ToPort: 5432
+      SourceSecurityGroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+
+  # Enable ingress from Secrets Manager.
+  {{ service.prefix }}SecretsManagerIngress:
+    Metadata:
+      'aws:copilot:description': 'Allow ingress from Secrets Manager in my application to the DB cluster'
+    Type: AWS::EC2::SecurityGroupIngress
+    Properties:
+      Description: Ingress from Secrets Manager
+      GroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+      IpProtocol: tcp
+      FromPort: 443
+      ToPort: 443
+      SourceSecurityGroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+
+  # Enable egress from Lambda Function.
+  {{ service.prefix }}LambdaEgress:
+    Metadata:
+      'aws:copilot:description': 'Allow egress from Lambda functions in my application to the DB cluster'
+    Type: AWS::EC2::SecurityGroupEgress
+    Properties:
+      Description: Egress from Lambda Function
+      GroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+      IpProtocol: tcp
+      FromPort: 5432
+      ToPort: 5432
+      DestinationSecurityGroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+
+  # Enable egress from Secrets Manager.
+  {{ service.prefix }}SecretsManagerEgress:
+    Metadata:
+      'aws:copilot:description': 'Allow egress from Secrets Manager in my application to the DB cluster'
+    Type: AWS::EC2::SecurityGroupEgress
+    Properties:
+      Description: Egress from Secrets Manager
+      GroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+      IpProtocol: tcp
+      FromPort: 443
+      ToPort: 443
+      DestinationSecurityGroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+
+  # Enable egress for HTTPS.
+  {{ service.prefix }}HTTPSEgress:
+    Metadata:
+      'aws:copilot:description': 'Allow egress for HTTPS'
+    Type: AWS::EC2::SecurityGroupEgress
+    Properties:
+      CidrIp: 0.0.0.0/0
+      Description: Egress for HTTPS
+      GroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+      IpProtocol: tcp
+      FromPort: 443
+      ToPort: 443
+
+  # Enable egress for HTTP.
+  {{ service.prefix }}HTTPEgress:
+    Metadata:
+      'aws:copilot:description': 'Allow egress for HTTP'
+    Type: AWS::EC2::SecurityGroupEgress
+    Properties:
+      CidrIp: 0.0.0.0/0
+      Description: Egress for HTTP
+      GroupId: !Ref '{{ service.prefix }}DBClusterSecurityGroup'
+      IpProtocol: tcp
+      FromPort: 80
+      ToPort: 80
```

Comment:
DO we need HTTPS and HTTP, or is HTTPS enough?

Reply:
Not sure - I will test this out!

Reply:
HTTPS is enough! Resolved in https://github.com/uktrade/copilot-tools/pull/185/commits/a385eceb3ab4c605beb82f257a738a8b87386a51.